pagopa/pn-frontend
# Support page - Feat/pn 10040

## Short description
Added support page for those users that are logged in.

## List of changes proposed in this pull request
- Added SupportPage component
- Added assistenza path to the routes
- Updated the behaviour of the support link (in the header) to manage the user logged case
- Created a utility file to manage the different app layouts
- Added new localization file
- Added new configuration for the support privacy policy link
- Created validator for the support page
- Added new test cases

## How to test
- got to pf portal as a not logged in user -> click on support link -> check that email provider is opened
- go to pf portal as logged in user -> click on support link -> check that new support page is shown -> submit the form and check that support form is opened
